tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java

```diff
@@ -7473,4 +7492,40 @@ public void uploadLatestResourceVersion(){
 		}
 	}
 
+	/**
+	 * To add twitter timeline with given parameters in a Lessons page
+	 */
+	public String addTwitterTimeline(){
+		//if username is not provided return
+		if(StringUtils.isBlank(twitterUsername)){
+			return "failure";
+		}
+		//if user has added @ symbol with the username, remove it
+		if( twitterUsername.contains("@")){
+			twitterUsername = StringUtils.remove(twitterUsername, "@");
+		}
+		String href  = "https://twitter.com/" + twitterUsername;
+		//Note: widget id used is from weblearn's twitter account
```

Comment:
Yep.

On 16 March 2016 at 10:21, Neelam Bhandari <notifications@github.com> wrote:

> In
> tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java
> <https://github.com/ox-it/wl-lessonbuilder/pull/22#discussion_r56310058>:
>
> > @@ -7473,4 +7492,40 @@ public void uploadLatestResourceVersion(){
> >  		}
> >  	}
> >
> > +	/**
> > +	 * To add twitter timeline with given parameters in a Lessons page
> > +	 */
> > +	public String addTwitterTimeline(){
> > +		//if username is not provided return
> > +		if(StringUtils.isBlank(twitterUsername)){
> > +			return "failure";
> > +		}
> > +		//if user has added @ symbol with the username, remove it
> > +		if( twitterUsername.contains("@")){
> > +			twitterUsername = StringUtils.remove(twitterUsername, "@");
> > +		}
> > +		String href  = "https://twitter.com/" + twitterUsername;
> > +		//Note: widget id used is from weblearn's twitter account
>
> yes you are right , this widget id will vary for each deployment of sakai
> as each institution will have different twitter account. shall i add this
> id into placeholder.properties for weblearn?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/ox-it/wl-lessonbuilder/pull/22/files/9344e82f34572b92df96e9a77aa9169972d0fed5#r56310058>
>



-- 
  Matthew Buckett, VLE Developer, IT Services, University of Oxford